### ISSUE TITLE
'Intune Connector for Active Directory' Text in not defined as heading in windows.

### ISSUE THREAD

**Issue by msftedad** [wix/react-native-navigation#8110]
### What happened?

**User Impact:**
Blind or low vision users may face significant accessibility barriers if the screen reader doesn't announce the heading information on 'Intune Connector for Active Directory' text. This can disrupt comprehension and workflow efficiency.

**Pre- Requisite:**
Turn on narrator.

**Actual Result:**
 'Intune Connector for Active Directory' Text in not defined as heading in windows.

**Refer Attachment:**
'Intune Connector for Active Directory' Text in not defined as heading in windows.png
'Intune Connector for Active Directory' Text in not defined as heading in windows.mp4

<img width="1024" height="628" alt="Image" src="https://github.com/user-attachments/assets/e48a7902-c6f7-4a61-87a8-25ba3260728f" />
https://github.com/user-attachments/assets/c9906221-295e-4a07-88a4-bb289b1b660d

### What was the expected behaviour?

'Intune Connector for Active Directory' Text should be defined as heading level 2 in windows.

### Was it tested on latest react-native-navigation?

- [x] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### Help us reproduce this issue!

1. Connect to the virtual machine.
2. Open 'ODJ Connector Bootstrapper' application.
3. 'Intune Connector for Active Directory dialog will be opened.
4. Press 'H' key to navigate the 'Intune Connector for Active Directory' dialog in scan mode.
5. Observe whether 'Intune Connector for Active Directory' Text in defined as heading in windows or not.

### In what environment did this happen?

OS: Windows 11 24H2 (OS Build 26100.4946)
App: ODJ Connector Bootstrapper
Screen Reader: Narrator

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
